src/main.ts

```diff
@@ -5,16 +5,8 @@ import { AppModule } from './app/app.module';
 import { environment } from './environments/environment';
 
 if (environment.production) {
-	enableProdMode();
+  enableProdMode();
 }
 
-// Uncomment the following lines to activate the service worker
```

Comment:
Why remove the service worker instruction?

Reply:
RESOLVED